Changes.md

```diff
@@ -1,6 +1,11 @@
 1.5.x.x (relative to 1.5.12.0)
 =======
 
+Features
+--------
+
+- SceneEditor : Added source scene tracking when viewing images. If Gaffer is able to determine which scene an image was rendered from ( an InteractiveRender to a Catalogue, or an ImageReader loading an image with metadata indicating it was rendered from a node in the current script ), then any SceneEditors will show the contents of the matching scene. This means that the Hierarchy View and Scene Inspector will show the hierarchy of the matching scene for the image, instead of being blank.
```

Comment:
I'm not 100% sure how I feel about this feature. On the one hand, it seems convenient. But on the other, I worry that it being automatic might make it seem more magical than it is, with it not being totally clear that the scene you're seeing is the scene that the source node happens to be outputting at the moment, rather than the one that was rendered (if contexts are different for instance).

Perhaps the main problem is that the node name doesn't appear in the editor at all when doing this (except for the SceneInspector), so you don't really see where the scene is coming from? Perhaps we could override `SceneEditor._titleFormat()` to always include the node name when we're doing this?

@murraystevenson, would be interested to know your thoughts.